working/0093 - Gradual Language Change Migration/0094 - Per Library Language Version Selection/strawman.md

```diff
@@ -50,6 +60,16 @@ library foo with 2.2;
 
 ```
 
+**Bob: This looks nice, but to the lexer, that's a floating point literal. It would be Very Bad if an implementation couldn't tell "2.2" and "2.20" apart. Should we quote it "2.2" just to be safe?**
```

Comment:
Or we could just decide that language level versions are integers, which would also make it even more obvious that it's not the same thing as the pubspec SDK constraint.